php/libraries/NDB_Reliability.class.inc

```diff
@@ -151,13 +151,22 @@ class NDB_Reliability extends NDB_Form
      * Attempts to validate the form (using the defined rules) and
      * saves the validated data into the database
      *
-     * @return void
+     * @return bool
      * @access public
      */
     function save()
     {
-        if ($this->form->validate()) {
-            $success = $this->form->process(array(&$this, "_save"), false);
+        if (!$this->form->validate()) {
+            return false;
+        }
+        $success = $this->form->process(array(&$this, "_save"), false);
+        if (is_bool($success)) {
+            return $success;
+        } else {
+            throw new Exception(
```

Comment:
LorisException.. but I just noticed the last 3 places I've made this comment look like the same code.... are there any differences, or should `save()` just be moved into the parent class?